win32ss/gdi/eng/pdevobj.c

```diff
@@ -336,6 +337,55 @@ PDEVOBJ_pSurface(
     return ppdev->pSurface;
 }
 
+BOOL
+PDEVOBJ_bEnableDirectDraw(
+    _Inout_ PPDEVOBJ ppdev)
+{
+    PGD_DXDDENABLEDIRECTDRAW pfnDdEnableDirectDraw = (PGD_DXDDENABLEDIRECTDRAW)gpDxFuncs[DXG_INDEX_DxDdEnableDirectDraw].pfn;
+    BOOL assertVal;
```

Comment:
why called "assertVal"? Maybe just "Success" ?

Reply:
I decided to name it `assertVal` based on other code style in this file. Look at other routines those are calling driver functions. Should I rename them to something like `Success` too? Definitely not in this PR.

Reply:
Okay, changed the variable name to `Success`, but only in this function.